doc/tutorials/acrn_on_qemu.rst

```diff
@@ -138,14 +140,14 @@ Install ACRN Hypervisor
 
 #. Install the ACRN build tools and dependencies following the :ref:`gsg`.
 
-#. Switch to the ACRN hypervisor ``v2.7`` tag.
+#. Switch to the ACRN hypervisor ``v3.0`` tag.
 
    .. code-block:: none
 
       cd ~
       git clone https://github.com/projectacrn/acrn-hypervisor.git
       cd acrn-hypervisor
-      git checkout v2.7
+      git checkout v3.0
```

Comment:
Line 156 uses the command `make BOARD=qemu SCENARIO=shared`. Does this command work for all boards? Should the user need to use the Configurator UI?

Reply:
This tutorial for running ACRN on the QEMU emulator, is only for the "qemu" board and not other target hardware.  As such the board XML and shared scenario XML files are supplied from the repo (misc/config_tools/data/qemu) and not generated by the board inspector or configurator tools.  (You could run the configurator and import the qemu.xml and shared.xml files into a new working folder to see what's the scenario contains.)

Reply:
@dbkinder What do you think about adding a sentence or two explaining this in the intro?